pkg/virt-handler/device-manager/usb_device.go

```diff
@@ -605,6 +605,11 @@ func discoverAllowedUSBDevices(usbs []v1.USBHostDevice) map[string][]*PluginDevi
 	localDevices := discoverLocalUSBDevicesFunc()
 	for _, usbConfig := range usbs {
 		resourceName := usbConfig.ResourceName
+		if usbConfig.ExternalResourceProvider {
+			log.Log.V(4).Infof("Skipping discovery of %s. To be handled by other device-plugin",
```

Comment:
nit: "by other" -> "by external".
I would not be opposed to increasing the verbosity even more.

Reply:
I'll move from 4 to 6, but we do need to define some [convention](https://github.com/kubevirt/kubevirt/issues/8164) eventually.

Thanks for the quick review

Reply:
I think such guidance would be nice.